packages/traceloop-sdk/traceloop/sdk/tracing/manual.py

```
@@ -66,10 +67,11 @@ def report_usage(self, usage: LLMUsage):
 
 
 @contextmanager
-def track_llm_call(vendor: str, type: str):
+def track_llm_call(vendor: Union[str, LLMVendor], type: str):
+    vendor_str = str(vendor)
     with get_tracer() as tracer:
-        with tracer.start_as_current_span(name=f"{vendor}.{type}") as span:
-            span.set_attribute(SpanAttributes.LLM_SYSTEM, vendor)
+        with tracer.start_as_current_span(name=f"{vendor_str}.{type}") as span:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)